### ISSUE TITLE
[1.3 modules beta2] Wildcard import doesn't work ?

### ISSUE THREAD

**Issue by vjau** [meteor/meteor#5872]
When doing ```import * as React from "react"``` i get following error in the browser console : ```Uncaught TypeError: babelHelpers.interopRequireWildcard is not a function```

**Comment by avital** [comment]
Copying over @trusktr's more complete stack frace from #5874:


```js
Uncaught TypeError: babelHelpers.interopRequireWildcard is not a function
  meteorInstall.app.src.client.config.routes.utilities.js @ utilities.js:5
  fileEvaluate                                            @ install.js:183
  require                                                 @ install.js:75
  meteorInstall.app.src.client.config.routes.index.js     @ index.js:6
  fileEvaluate                                            @ install.js:183
  require                                                 @ install.js:75
  (anonymous function)                                    @ app.js?hash=3cc4b471ccc6efbb3976924b77fd96ea30aa0190:32042
  (anonymous function)                                    @ app.js?hash=3cc4b471ccc6efbb3976924b77fd96ea30aa0190:32077
```

**Comment by fermuch** [comment]
this is specially useful when importing lodash.

Outside meteor, I usually do:

```javascript
import * as _ from 'lodash';
```

**Comment by trusktr** [comment]
Why not `import _ from 'lodash'` in this case?

**Comment by fermuch** [comment]
for some weird reason, webpack+babel throws an error if I try to do that. May be a problem with my setup, tho'.

**Comment by rozzzly** [comment]
What build of lodash are you using? Sounds like that's the issue there

**Comment by vjau** [comment]
No, the issue is to support wildcard imports, which are from the standard.
Le 18 janv. 2016 21:57, "Patrick Lienau" <notifications@github.com> a
écrit :

**Comment by benjamn** [comment]
I believe this was fixed (in Meteor, at least) by https://github.com/meteor/meteor/commit/3a9907aa3acbee84aaa5759bfcb241bf1fe8b229.

**Comment by vjau** [comment]
Confirmed fixed in the checkout, not in the beta release. 
Thanks !

**Comment by benjamn** [comment]
@avital and I will do a bug triage tomorrow, and if there are no show-stopping new bugs we still need to fix, we'll release another beta with this fix (and several others)

**Comment by vjau** [comment]
Confirmed fixed in 1.3 modules beta 5
